queries.sql

```diff
@@ -8,3 +8,100 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name !='Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3; 
 
+-- Start Transaction --
```

Comment:
- [ ] [optional] @josuerushanika great work on your queries, however, it would have been nice if your screenshot were taken in a better way showing not only the result but also the query command

Reply:
@danmainah Thank you for your review,